### PR TITLE
Make Dept Head Checklist emails sendable in POST_CON mode

### PR DESCRIPTION
**Pull request by kitsuta**
We have a department head checklist email that's meant to be sent after the event, but refuses to send any emails because we're in post_con mode now. This enables all the emails during post con mode.

### CODE REVIEW

**Comment by binary1230** [comment]
actually this won't quite do it, explanation:

**Comment by kitsuta** [comment]
Merging now, since people are waiting on this and we've already discussed this change over Slack. :)

**Comment by binary1230** [comment]
short version is, that will force ALL dept head checklist emails to only be able to send when c.POST_CON mode is true


[4:41] 
which, we want that to not be true for stuff like "remember to sign up for shifts"

**Comment by RobRuana** [comment]
We need to update puppet, config.py, automated_emails.py, and production-config.

**Comment by RobRuana** [comment]
Add a post_con setting to the dept checklist, and only set it to true for the last item.

**Comment by RobRuana** [comment]
That was very cursory. Did that make sense?

We should just do it via ssh/vi for west and labs, then we'll fix the whole thing in reggie.

File: uber/automated_emails_server.py [hunk 1]
```diff
@@ -322,7 +322,8 @@ def __init__(self, conf):
                                 ident='department_checklist_{}'.format(conf.name),
                                 when=days_before(7, conf.deadline),
                                 sender=c.STAFF_EMAIL,
-                                extra_data={'conf': conf})
+                                extra_data={'conf': conf},
+                                post_con = conf.email_post_con or False)
```
Comment (kitsuta) [inline uber/automated_emails_server.py]: OK, the `or False` is probably unnecessary, but still.

**Comment by binary1230** [comment]
+1 if the falsey logic works (which it looks like it would)

**Pull request merged**